docs/guides/_toc.json

```diff
@@ -185,7 +185,7 @@
                   "title": "OpenQASM",
                   "children": [
                     {
-                      "title": "Intro to OpenQASM",
+                      "title": "Introduction to OpenQASM",
```

Comment:
If we change it in the toc we need also to change the title on the file itself. The title in the toc should match the title on the page. I will suggest that change further down